app/controllers/episode_media_controller.rb

```diff
@@ -49,18 +49,22 @@ def update
         format.html { render :show, status: :unprocessable_entity }
       end
     end
+  rescue ActiveRecord::StaleObjectError
+    render :show, status: :conflict
   end
 
   private
 
   def set_episode
     @episode = Episode.find_by_guid!(params[:episode_id])
     @episode.strict_validations = true
+    @episode.locking_enabled = true
```

Comment:
Yeah, I did try it that way, but discovered it really needs to default to `false`. Otherwise all the `belongs_to ... touch: true` throughout the app will lock all the parent records as updated_ats bubble up.